ocfweb/docs/docs/staff/backend/switch.md

```diff
@@ -1,15 +1,22 @@
 [[!meta title="Managed switches"]]
 
-We use an [Arista 7050SX-64][primary-switch] 10GbE switch as a primary switch
-for our servers, and two [Arista 7048T-A][secondary-switch] 1GbE switches for
+We use an [Arista 7050S-64][primary-switch] 10GbE switch as a primary switch
+for our servers, and two [Arista 7048T-4S][secondary-switch] 1GbE switches for
 desktops and management respectively. These devices were donated to us by
-Arista Networks in Fall 2018. Each device in the lab connects first to the back
+Arista Networks in Fall 2018. 
+
+The Arista 7050S-64 is a Trident+ based switching platform, featuring 48 10G-mode
+ports, and 4 40G-mode ports. Each of the SFP+ cages on the 7050S-64 can support 10G and 1G mode operation.  
+The QSFP ports can be channelized out to 4x10G in order to gain a total of 64 10G ports in dire situations.  However,
+keep in mind that changing the QSFP+ ports from 10G to 40G will cause all the interfaces to flap (including the management interface!) -- so schedule accordingly.
+
+Each device in the lab connects first to the back
 of a patch panel, and then from a port on the patch panel to a port on one of
-the 7048T-As via Cat6. The servers connect directly to the 7050SX via SFP+ DACs,
-as does our uplink to IST, through a Cat6 to 1GbE SFP+ converter. The 7048T's
-connect to the 7050SX through their SFP+ uplink ports.
+the 7048T-4S' via Cat6. The servers connect directly to the 7050S-64 via SFP+ DACs,
+as does our uplink to IST, through a RF45 to 1GbE SFP module. The 7048T's
+connect to the 7050S-64 through their SFP+ uplink ports.
 
-We do not currently use many of the managed features of the switches, mostly using
+We do not currently use any of the managed features of the switches, mostly using
```

Comment:
Is this true now? I thought we now dropped IPv6 RAs again because we were having issues there with instances auto-configuring themselves with unroutable addresses from the university.

Reply:
I don't think the RA's are dropped on the switch layer, rather the host.  I will confirm when I get a chance

Reply:
they are not dropped at the switch layer.